### DISCUSSION TITLE
Homebridge Bond Broken after 3.2.2

### DISCUSSION THREAD

**Discussion by jbyerline** [aarons22/homebridge-bond#124] [General]
After updating I am getting errors in the homebridge log. The Bond app still works and the fans all show up in the home app but the home app no longer communicates with the fans. 

> [18/12/2020, 23:23:03] [Bond] 8 cached accessories were loaded
> [18/12/2020, 23:23:03] [Bond] Getting devices for this Bond (BD21401)...
> [18/12/2020, 23:23:03] [Bond] 4 devices were found on this Bond (BD21401).
> [18/12/2020, 23:23:03] [Bond] No new devices to add for this Bond (BD21401).
> [18/12/2020, 23:23:04] [Bond] Configuring Accessory: Jacob’s Room Jacob's Fan
> [18/12/2020, 23:23:04] [Bond] No Bond found for Device: Jacob's Fan.
>           This Device may have been removed from your Bond but still exists in cachedAccessories.
> [18/12/2020, 23:23:04] [Bond] Configuring Accessory: Master Bedroom Master Fan
> [18/12/2020, 23:23:04] [Bond] No Bond found for Device: Master Fan.
>           This Device may have been removed from your Bond but still exists in cachedAccessories.
> [18/12/2020, 23:23:04] [Bond] Configuring Accessory: Office Michael's Fan
> [18/12/2020, 23:23:04] [Bond] No Bond found for Device: Michael's Fan.
>           This Device may have been removed from your Bond but still exists in cachedAccessories.
> [18/12/2020, 23:23:04] [Bond] Configuring Accessory: Sarah’s Room Sarah's Fan
> [18/12/2020, 23:23:04] [Bond] No Bond found for Device: Sarah's Fan.
>           This Device may have been removed from your Bond but still exists in cachedAccessories.
> [18/12/2020, 23:23:04] [Bond] Configuring Accessory: Jacob’s Room Jacob's Fan
> [18/12/2020, 23:23:04] [Bond] [Jacob's Fan] actions: TurnOff,StartDimmer,SetSpeed,ToggleLight,Stop,TogglePower,TurnOn,IncreaseSpeed,DecreaseSpeed,TurnLightOn,TurnLightOff
> [18/12/2020, 23:23:04] [Bond] Configuring Accessory: Master Bedroom Master Fan
> [18/12/2020, 23:23:04] [Bond] [Master Fan] actions: ToggleLight,TurnOff,StartDimmer,SetSpeed,Stop,TogglePower,TurnOn,IncreaseSpeed,DecreaseSpeed,TurnLightOn,TurnLightOff
> [18/12/2020, 23:23:04] [Bond] Configuring Accessory: Office Michael's Fan
> [18/12/2020, 23:23:04] [Bond] [Michael's Fan] actions: SetSpeed,ToggleLight,TurnOff,Stop,TogglePower,TurnOn,IncreaseSpeed,DecreaseSpeed,TurnLightOn,TurnLightOff
> [18/12/2020, 23:23:04] [Bond] Configuring Accessory: Sarah’s Room Sarah's Fan
> [18/12/2020, 23:23:04] [Bond] [Sarah's Fan] actions: StartDimmer,SetSpeed,ToggleLight,TurnOff,Stop,TogglePower,TurnOn,IncreaseSpeed,DecreaseSpeed,TurnLightOn,TurnLightOff

**Comment by aarons22** [comment]
Any idea what your previous version was?
